guides/getting_started_command_line.rst

```diff
@@ -29,6 +31,14 @@ an RPi, please install ESPHome through ``pip`` or use :doc:`the Hass.io add-on <
 
     docker pull esphome/esphome
 
+Installation on Windows 10
+--------------------------
+
+It's possible to also install ESPHome command line in Windows 10, inside the `Windows Subsystem for Linux
+(WSL) <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__ using the same ``pip`` command as
+above. The main limitation is that COM ports are not exposed properly in WSL, so only OTA flashing works.
+Or local flashing from Windows using `esphome-flasher <https://github.com/esphome/esphome-flasher>`__.
```

Comment:
Hmm, I don't think this is important enough to put in the main getting started guide. This guide should be bare minimum with no big tangents for specific setups.

How about moving this to the FAQ and linking that from here?